app/models/card.rb

```diff
@@ -16,4 +16,13 @@ def check_duplication
   def set_default_date
     self.review_date = Date.today + 3.days
   end
+
+  def compare_translations(submitted_text)
+   if self.translated_text.mb_chars.downcase.to_s == submitted_text.mb_chars.downcase.to_s
```

Comment:
Line is too long. [90/80]<br>Use 2 (not 1) spaces for indentation.<br>Redundant `self` detected.